packages/payment-detection/test/eth/info-retriever.test.ts

```diff
@@ -45,60 +53,34 @@ describe('api/eth/info-retriever', () => {
     await expect(infoRetreiver.getTransferEvents()).rejects.toThrowError();
   });
 
-  // Utility for conditionally skipping tests
-  const describeIf = (
-    condition: any,
-    ...args: [string | number | Function | jest.FunctionLike, jest.EmptyFunction]
-  ) => (condition ? describe(...args) : describe.skip(...args));
+  it('can detect a MATIC payment to self', async () => {
+    // NB: The from and to are the same
```

Comment:
It might just be me but I always get tripped up by the NB acronym for Nota Bene. I think "Note" would be more clear.

```suggestion
    // Note: The from and to are the same
```